docs: Add documentation style guidelines (#141)
coconut-svsm/svsm: README.md

Contributing to the project is as easy as sending a pull-request via
GitHub. For detailed instructions on patch formatting and contribution

> guidelines please have a look at [CONTRIBUTING.md](CONTRIBUTING.md) and
> for documentation guidelines to [DOC-GUIDELINES.md](Documentation/DOC-GUIDELINES.md).

Reviewer: ```suggestion
guidelines please have a look at [CONTRIBUTING.md](CONTRIBUTING.md).
For documentation guidelines consult [DOC-GUIDELINES.md](Documentation/DOC-GUIDELINES.md).
```

Author: Done!
